server/controllers/entryController.js

```diff
@@ -0,0 +1,31 @@
+import datetime from 'node-datetime';
+import grabEmployeeIdFromToken from '../helpers/tokenDecoder';
+import ResponseHandler from '../helpers/responseHandler';
+import {
+  RESOURCE_CREATED, REQUEST_SUCCEDED, SERVER_ERROR, REQUEST_CONFLICT, NOT_FOUND,
+} from '../helpers/statusCode';
+import Database from '../models/database';
+
+class EntryController {
+    static createEntry = async (req, res) => {
+      try {
+        let { title, description } = req.body;
+        let token = req.header('x-auth-token');
+        const owner = grabEmployeeIdFromToken(token, res);
+        const findIfEntryexist = 'SELECT * FROM entries WHERE title = $1 and ownerid = $2';
+        const entryExist = await Database.execute(findIfEntryexist, [title, owner]);
+        if (entryExist[0]) {
+          return ResponseHandler.error(REQUEST_CONFLICT, 'The same entry exists', res);
+        }
```

Comment:
@murera I think we don't need to check if a title already exist, as long as we have two different IDs